operator/gitops/argocd/pipeline-service/openshift-pipelines/appstudio-pipelines-runner-clusterrole.yaml

```diff
@@ -0,0 +1,49 @@
+---
+# Any change to this role must be communicated to the devsandbox team
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: appstudio-pipelines-runner-clusterrole
```

Comment:
cc @Michkov @stuartwdouglas - I think the app studio build definitions need more permissions than what are listed here.